app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= provide(:title, "About") %>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield(:title)%> | Attinderance </title>
+  </head>
+  <body>
+    <h1>Attinderance | <%= yield(:title)%></h1>
+    <br>
+    <p>Attinderance was originally intended to be a RoR learning project with real-world application. As a college instructor, I wanted a quick and easy way to take attendance that was unobtrusive. Since my students are adults, calling out the rollcall seemed akward and offensive. Yet for scholarship and admin purposes, attendance was useful and in cases mandatory.</p>
+    <br>
+    <p>Railsbridge Boston, a workshop set up by volunteers from the Boston Rails community, inspired me to learn the framework and Ruby language. This app seemed like a good independent project. Of course, I couldn't do this myself.</p>
+    <br>
+    <p>Thanks go to: my mentor, Edward Loveall; Railsbridge oranizer, Melissa Xie, and BostonRubyWomen MeetUp </p>
+    <br>
+<h2>Site Map</h2>
+<p><%= render '/applications/side_bar' %></p>
```

Comment:
For what it's worth, when I changed it to `render 'side_bar'`, I saw this error message:

```
Missing partial static_pages/_side_bar, application/_side_bar
```

which led me to see `application` instead of `applications`